apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/persistence/RestApiV2QMaterializedViewIT.java

```diff
@@ -81,9 +81,9 @@ public void getRowsFromMV() throws Exception {
     // 14-Sep-2022, tatu: Not sure why but there is a transient issue here wrt timing.
     //   Locally test does not appear to ever fail, but in CI it does, with error suggesting
     //   MV has not been created or Schema metadata not (yet) updated. In absence of
-    //   better solution, let's try simple wait to give it time; 5 seconds should do it.
+    //   better solution, let's try simple wait to give it time; 3 seconds should do it.
 
-    Thread.sleep(5000L);
+    Thread.sleep(3000L);
```

Comment:
better than `Thread.sleep` is to use `Awaitability`, there should be several examples on usage in docs api..

Reply:
Yes if there was a way to check for condition. Would love to use it but here all I get is weird failure (index not yet created etc, or whatever preceding query tried to create)

Reply:
I guess I can change it for this particular case tho as query tested is not too complex.

Reply:
Thank you @ivansenic I forgot it's fine to just have the thing fail and not necessarily have condition to wait. Changed as suggested, thanks!